server/parameters/ServerSubcommandParser.java

```diff
@@ -77,7 +77,7 @@ public Server(ConfigParser configParser) {
             Optional<Path> configPath = Server.configPath.parse(auxOptions);
             Config config = configPath
                     .map(path -> ConfigFactory.config(path, configOptions, configParser))
-                    .orElse(ConfigFactory.config(configOptions, configParser));
+                    .orElseGet(() -> ConfigFactory.config(configOptions, configParser));
```

Comment:
This was not lazy before, and could fail if the default config path did not exist!